aes_cmac/aes_cmac.c

```diff
@@ -25,27 +25,11 @@
 #include "aes_cmac.h"
 #include "../common/insecure_memzero.h"
 
-static uint8_t zero[AES_BLOCK_SIZE];
-
-/*#include <stdio.h>
-static void dump_hex(char *msg, const unsigned char *buf, unsigned int len) {
```

Comment:
Any reason to remove och comment out this? It could be a good function to have when debugging...

Reply:
Only that it was not used. En general I don't think commented out code shod be present.